Build error starting with `2.1.0` on Windows (MingGW32 & Git Bash)

Picotool compiles and executes successfully in my build environment with version `2.0.0` (same SDK version).  Updating to `2.1.0` or `2.1.1` for both results in the following error when executing `make`:

```
C:/mingw32/bin/../lib/gcc/i686-w64-mingw32/14.2.0/../../../../i686-w64-mingw32/bin/ld.exe: C:/VSARM/sdk/pico/libusb/MinGW32/static/libusb-1.0.a(core.o):core.c:(.text+0x121): undefined reference to `_imp___iob'
```

I am curious if this could be related to PR #126?  Seems like that was a significant change to how libusb is referenced, and this error relates to what [Stack Overflow would suggest](https://stackoverflow.com/questions/66314384/undefined-reference-to-imp-in-minwg) might be static vs. dynamic linking.

For reference, my build environment is an updated version of what I outline in [this repo](https://github.com/recursivenomad/capacitance-measurement-module/blob/main/BUILDING.md).  Note that the versions linked there are outdated, but the process remains the same.

TLDR: I use Git Bash as my terminal, with a manually installed MinGW32.  I have a hunch that linking against the static `libusb-1.0.a` has broken my build, but I am unsure of the appropriate solution.

Let me know if I can provide any further information :)

--- a/cmake/FindLIBUSB.cmake
+++ b/cmake/FindLIBUSB.cmake
@@ -59,9 +59,11 @@ else (LIBUSB_INCLUDE_DIR AND LIBUSB_LIBRARIES)
     include(FindPackageHandleStandardArgs)
     FIND_PACKAGE_HANDLE_STANDARD_ARGS(LIBUSB DEFAULT_MSG LIBUSB_LIBRARIES LIBUSB_INCLUDE_DIR)
 
-    # Don't use .dll.a libraries, as they require the .dll file to be in the correct location
-    # Replace with .a for static linking instead
-    string(REPLACE ".dll.a" ".a" LIBUSB_LIBRARIES ${LIBUSB_LIBRARIES})
+    if (NOT PICOTOOL_LIBUSB_ALLOW_DLL)
+        # Don't use .dll.a libraries, as they require the .dll file to be in the correct location
+        # Replace with .a for static linking instead
+        string(REPLACE ".dll.a" ".a" LIBUSB_LIBRARIES ${LIBUSB_LIBRARIES})
+    endif()
 
     MARK_AS_ADVANCED(LIBUSB_INCLUDE_DIR LIBUSB_LIBRARIES)
 endif (LIBUSB_INCLUDE_DIR AND LIBUSB_LIBRARIES)

